### PR TITLE
shows a message when the search returns no content

### PR DESCRIPTION
**Pull request by shubhsingh594**
Fixes https://github.com/learningequality/ka-lite/issues/4927

### CODE REVIEW

File: kalite/main/api_views.py [hunk 1]
```diff
@@ -34,6 +34,9 @@ def search_api(request, channel):
 
     matches, exact, pages = search_topic_nodes(query=query, channel=channel, language=request.language, page=1, items_per_page=15, exact=False)
 
+    if matches==list():
```
Comment (rtibbles) [inline kalite/main/api_views.py]: A more Pythonic way to do this would be to test matches for emptiness, rather than seeing if it equals an empty list.

In Python empty objects are False-y, so we could just do:
```
if not matches:
```

**Comment by rtibbles** [comment]
:+1:
